tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@tsconfig/node14/tsconfig.json",
+  "compilerOptions": {
+    "module": "ES2020",
+    "moduleResolution": "node"
```

Comment:
I get a ts error when I import an `.mjs` file without a declaration file in an `.mts` file. Should we allow JS files to be imported using the `allowJs: true` compiler option?

Reply:
done